lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -48,40 +44,22 @@ def non_threadsafe_bulk(actions)
         else
           next { action => args }
         end
-      end.flatten
-
-      @client.bulk(:body => bulk_body)
-    end
-
-    def start_sniffing!
-      if options[:sniffing]
-        @sniffer_thread = Thread.new do
-          loop do
-            @request_mutex.synchronize { sniff! }
-            sleep (options[:sniffing_delay].to_f || 30)
-          end
-        end
+      end.
+      flatten.
+      reduce("") do |acc,line|
```

Comment:
Not really possible with this format since we need the newlines in specific places.